@here/harp-olp-utils/lib/OlpCopyrightProvider.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AreaCopyrightInfo, CopyrightCoverageProvider } from "@here/harp-mapview";
+import { CatalogClient, DataStoreContext, HRN } from "@here/olp-sdk-dataservice-read";
+
+/**
+ * `OlpCopyrightProvider` initialization parameters.
```

Comment:
[[OlpCopyrightProvider]] for better UX in typedoc

Reply:
Done.